src/Form/RedirectFix404Form.php

```diff
@@ -66,54 +66,62 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       );
     }
 
+    $languages = \Drupal::languageManager()->getLanguages();
```

Comment:
What happens with this page if the feature is disabled? I think we should have an access check that only shows it if that feature is enabled? Or should we maybe implement it so that we use our table only optionally if the setting is enabled and otherwise stick to the watchdog table?